lib/actions/term-groups.js

```diff
@@ -171,3 +173,19 @@ export function exitActiveTermGroup() {
     });
   };
 }
+
+export function requestPaneMaximizedStateToggle() {
+  return (dispatch, getState) => {
+    dispatch({
+      type: TERM_GROUP_TOGGLE_MAXIMIZED,
+      effect() {
+        const {sessions, termGroups} = getState();
+        const {uid} = findBySession(termGroups, sessions.activeUid);
+        dispatch({
```

Comment:
> Finding the active TermGroup to maximize should be done on the reducer, I think.

I don't have access to `state.sessions` inside `termGroups` reducer, so it's easier to use thunk for it. Also this pattern is used in many other action creators (incl `moveToNextPane`).

> Two actions for this feature seems to much no ?

I agree in general. Actually I did it this way b/c of the `effect` method, if it's not necessary here, I can refactor to single action.

> Implementation of this feature should be like moveToNextPane() action, imo.

Another way is to use `MAXIMIZE` / `RESTORE_MAXIMIZED` actions instead of `TOGGLE`, leaving it up to reviewers.